$NetBSD: patch-ac,v 1.1 2006/07/02 09:53:00 rillig Exp $

Fixed the way the pathname of the configuration file is built up.

--- lib/read_config.c.orig	2003-10-26 04:36:51.000000000 +0100
+++ lib/read_config.c	2006-07-02 09:02:07.000000000 +0200
@@ -52,8 +52,8 @@ int read_config(char *fname, char *secti
  int bad = NO, line_ct = 0, res;
 
  if(fname == NULL) {
-  snprintf(buf, STRBUF, "%s/%s", ubs_table_data(&GLOBAL, "prefix"), DEF_CONFIG);
-  if((config_fp = fopen(buf, "r")) == NULL) {
+  fname = DEF_CONFIG;
+  if((config_fp = fopen(fname, "r")) == NULL) {
    return NO_FILE;
   }
  }
